src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java

```diff
@@ -37,7 +38,7 @@
     implements FileSetCheck {
 
     /** Collects the error messages. */
-    private final LocalizedMessages messageCollector = new LocalizedMessages();
+    private final SortedSet<LocalizedMessage> errorMessages = new TreeSet<>();
```

Comment:
why rename this to `errorMessages` (specifically `error`) when you want it to be `violation` at #4465 ?